apps/client/src/common/components/navigation-menu/FloatingNavigation.tsx

```diff
@@ -39,22 +44,30 @@ export default function FloatingNavigation(props: FloatingNavigationProps) {
 
   return (
     <div className={`${style.buttonContainer} ${!showButton ? style.hidden : ''}`}>
-      <button
-        onClick={toggleMenu}
-        aria-label='toggle menu'
-        className={style.navButton}
-        data-testid='navigation__toggle-menu'
-      >
-        <IoApps />
-      </button>
-      <button
-        className={style.button}
-        onClick={toggleSettings}
-        aria-label='toggle settings'
-        data-testid='navigation__toggle-settings'
-      >
-        <IoSettingsOutline />
-      </button>
+      {isViewLocked ? (
```

Comment:
I am not sure we can do it globally here
we have two Navigation implementations
`ProductionNavigationMenu` and `ViewNavigationMenu`

The feature here is applicable to the views that consume `ViewNavigationMenu`, I suggest that we encapsulate this logic there
this also allows us some DOM cleanups, since we know that we wont navigate, we just need to show the <FloatingNavigation /> element containing our lock icon